apiservers/portlayer/swagger.yml

```diff
@@ -393,12 +393,12 @@ definitions:
         $ref: "#/definitions/ImageStore"
       image:
         type: string
-      cmd:
+      path:
```

Comment:
Tether uses that convention. Also Docker does something similar internally they drop cmd and start using entypoint and args (after merging cmd and entrypoint together) https://github.com/docker/docker/blob/master/daemon/exec.go#L91